test/e2e/adminUI/tests/group003List/uxTestListView.js

```diff
@@ -21,184 +21,145 @@ module.exports = {
 		browser.end();
 	},
 	'List view should allow users to create a new list item': function (browser) {
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
```

Comment:
Again, this should be a command really. We're trying not to access the PO elements, except via commands.